src/encoded/tests/test_audit_experiment.py

```diff
@@ -3137,4 +3137,4 @@ def test_is_control_dataset(testapp, control_target, ctrl_experiment, publicatio
     assert is_control_dataset(series_embedded) == True
     file_set = testapp.get(publication_data['@id'] + '@@index-data')
     file_set_embedded = file_set.json['embedded']
-    assert is_control_dataset(file_set_embedded) == False
\ No newline at end of file
+    assert is_control_dataset(file_set_embedded) == False
```

Comment:
This is an unfortunate but common unnecessary change.  Seems like some people's text editors add these while other's remove them.  

I get around this and many other similar issues by always using git add --patch.  It's weird to become accustom to, but a great tool that leads to more consistent coding practices.